superset-frontend/src/dashboard/components/nativeFilters/FilterBar/FilterControls/FilterControl.tsx

```diff
@@ -142,7 +158,24 @@ const HorizontalFormItem = styled(StyledFormItem)`
   }
 
   .ant-form-item-control {
-    width: ${({ theme }) => theme.gridUnit * 41}px;
+    width: ${({ theme, inverseSelection }) =>
+      inverseSelection ? theme.gridUnit * 41 * 1.6 : theme.gridUnit * 41}px;
```

Comment:
width should not use `gridUnit` which should be only meant for spacing. Can we use a fixed value here? Also, do we really need to set a specific width?